user/tests/wiring/no_fixture_stress/exflash.cpp

```diff
@@ -172,3 +205,17 @@ test(EXFLASH_02_rtl872x_validate_mode) {
     assertEqual((uint32_t)SPIC->baudr, (uint32_t)1);
 }
 #endif // HAL_PLATFORM_RTL872X
+
+test(EXFLASH_03_aligned_and_unaligned_writes_from_internal_flash_work) {
```

Comment:
What part of this test is `from_internal_flash`? Does gen3 execute this code as XIP? 
`hal_storage_write()` is the external OTA flash section for both platforms right?

Reply:
`static uint8_t dummy[8192 + 4] __attribute((aligned(4))) = {0xff, 0xff, 0xff, 0xff};` is internal on gen 3, RAM on Gen 4, but the test is still kind of valid for gen 4.

Reply:
Oops, that needs to be `static const` now that you mention it.